homeassistant/components/netgear_lte/notify.py

```diff
@@ -2,28 +2,23 @@
 import logging
 
 import attr
-import voluptuous as vol
 
 from homeassistant.components.notify import (
-    ATTR_TARGET, PLATFORM_SCHEMA, BaseNotificationService)
-from homeassistant.const import CONF_HOST
-import homeassistant.helpers.config_validation as cv
+    ATTR_TARGET, BaseNotificationService, DOMAIN)
 
-from ..netgear_lte import DATA_KEY
+from ..netgear_lte import CONF_RECIPIENT, DATA_KEY
```

Comment:
We usually write like this:
```py
from . import CONF_RECIPIENT, ...
```

Reply:
LOL – yeah, it seems the Great Migration missed that pattern.